webapp/packages/core-utils/src/formatNumber.ts

```diff
@@ -8,7 +8,7 @@
 
 export function formatNumber(n: number, d: number) {
   if (n < 1000) {
-    return n.toString();
+    return n.toLocaleString(undefined, { maximumFractionDigits: 2 });
```

Comment:
I think toFixed(2) will be enough

Reply:
toFixed will make 85.00 from 85, why do we need it?

Reply:
In case of the toLocaleString we depend on the user local machine setup, it will create different behaviour for users. But I think its not that important for now